specsms_confirm_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'text_alert'
+
+describe TextAlert do
+  # No tests for external API
```

Comment:
You can't test the API but you can test the methods using doubles. For reference here is once method (with your method names instead obvs): 

    `expect(@sms).to receive(:send_confirmation)
    @sms.send_confirmation(@time)`